pkg/model/iam/subject.go

```diff
@@ -83,11 +82,7 @@ func BuildNodeRoleSubject(igRole kops.InstanceGroupRole) (Subject, error) {
 
 // ServiceAccountIssuer determines the issuer in the ServiceAccount JWTs
 func ServiceAccountIssuer(clusterName string, clusterSpec *kops.ClusterSpec) (string, error) {
-	if featureflag.PublicJWKS.Enabled() {
-		return "https://api." + clusterName, nil
-	}
-
-	return "", fmt.Errorf("ServiceAcccountIssuer not (currently) supported without PublicJWKS")
+	return "https://api." + clusterName, nil
```

Comment:
Somewhat unrelated to what this PR is trying to fix, if the admin has set `spec.kubeAPIServer.serviceAccountIssuer` explicitly, this should probably return that value instead.

Reply:
I agree we should respect the API field but that could be done in a separate PR.

This function no longer needs to return an error